lib/DBIx/Class/Fixtures.pm

```diff
@@ -1402,7 +1402,13 @@ sub populate {
              $self->msg("- updating sequence $sequence");
             $rs->result_source->storage->dbh_do(sub {
               my ($storage, $dbh, @cols) = @_;
-              $self->msg(my $sql = "SELECT setval('${sequence}', (SELECT max($column) FROM ${table}));");
+              $self->msg(
+			 my $sql = "SELECT setval('${sequence}', (SELECT max("
```

Comment:
The sequence name needs to be quoted (or even better, passed in via a placeholder) too.